lib/tasks/etl_db_check.rake

```diff
@@ -0,0 +1,138 @@
+require 'net/http'
+require 'uri'
+
+namespace :etldb do
+    db_name = ENV['ETL_DB_NAME']
+    db_port = ENV['ETL_DB_PORT']
+    db_host = ENV['ETL_DB_HOST']
+    db_user = ENV['ETL_DB_USER']
+    db_pass = ENV['ETL_DB_PASSWORD']
+
+    tables = [
+        {
+            'title' => "Daily",
+            'name' => "daily_ohcl_prices",
+            'interval' => "''1 day''",
+            'url' => "https://hc-ping.com/772bacf5-958e-4541-b8ca-a81fe24c63d0"
+        },
+        {
+            'title' => "Hourly",
+            'name' => "hourly_ohcl_prices",
+            'interval' => "''1 hour''",
+            'url' => "https://hc-ping.com/c49dcf87-c6e5-4881-95f8-3bfad31d33ff"
+        }
+    ]
+
+    coins = [
+        {
+            'title' => "'Bitcoin",
+            'query' => lambda { Coin.where(symbol: 'BTC').pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "Ethereum",
+            'query' => lambda { Coin.where(symbol: 'ETH').pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "CoinFi",
+            'query' => lambda { Coin.where(symbol: 'COFI').pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "Top 10",
+            'query' => lambda { Coin.top(10).pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        },
+        {
+            'title' => "Random 3",
+            'query' => lambda { Coin.where("ico_status=? OR ranking IS NOT NULL", 'listed')
+                .order("RANDOM()").limit(3).pluck(:coin_key, :ranking) },
+            'url' => "https://hc-ping.com/67fffb49-c4d7-4e3f-a7cf-4688495c00d1"
+        }            
+    ]
+
+    desc "check for recent entries in specified table of etl database"
+    task :check, [:index] => :environment do |task, args|
```

Comment:
Can we add more supporting documentation or split this method up a bit. It's a bit hard to follow especially with 3 nested loops

Reply:
Added commenting. Let me know if it's still unclear.